roles/tools/defaults/main.yaml

```diff
@@ -8,9 +8,11 @@ tools_kubectl_required: false
 tools_kubectl_image: docker.io/bitnami/kubectl
 tools_kubectl_image_tag: 1.25
 
-tools_mongodb_conf: "{{ ansible_env.HOME }}/.mongorc.js"
+tools_mongodb_conf: "{{ ansible_env.HOME }}/.{{ tools_mongodb_cli }}.js"
 tools_mongodb_image: docker.io/percona/percona-server-mongodb
 tools_mongodb_install: "{{ 'mongodb' in groups and groups['mongodb'] | length > 0 }}"
+tools_mongodb_mongosh:
+  - 6.0
```

Comment:
@aaronplagge why is this only v6, is it completely backwards compatible? The docs seem to suggest getting it by installing MongoDB, which would then suggest that it should match the version

Reply:
That only lists v6 because only that image has the mongosh executable.  The prior versions only had mongo.  I left this as a list as future versions of the mongo container image will also only have mongosh.  Since we setup aliases to use the mongo container images I did not consider installing mongosh locally.  Either way the mongosh version that is in the v6 image works with prior versions of mongo.